tests/images/kafka.rs

```diff
@@ -0,0 +1,147 @@
+use futures::{
+    future::{self, join_all},
+    StreamExt, TryStreamExt,
+};
+use rdkafka::{
+    config::ClientConfig,
+    consumer::{stream_consumer::StreamConsumer, Consumer},
+    producer::{FutureProducer, FutureRecord},
+    Message,
+};
+use std::{collections::HashMap, time::Duration};
+use uuid::Uuid;
+
+use testcontainers::{
+    clients,
+    core::{RunArgs, WaitFor},
+    images::{kafka, zookeeper},
+    Image,
+};
+
+const ZOOKEEPER_PORT: u16 = 2181;
+
+/// Populates `topic` once a Kafka broker is up and running.
+fn create_topic(docker: &clients::Cli, network: &Uuid, topic: &str) {
+    let topic_creation_command = format!("cub kafka-ready -b broker:{} 1 20 && \
+kafka-topics --create --if-not-exists --bootstrap-server broker:{} --partitions 1 --replication-factor 1 --topic {}",
+	kafka::DEFAULT_BROKER_LISTENER_PORT, kafka::DEFAULT_BROKER_LISTENER_PORT, topic);
+
+    // Once `topic` has been created, it is ok to destroy the container at the end of the function
+    // because it is meant to be a one-shot.
+    docker.run_with_args(
+        kafka::Kafka::default()
+            .with_wait_for(WaitFor::message_on_stdout("Created topic"))
```

Comment:
Addressed in 0ae4d44. Originally, this `docker run` needed a different wait condition because this container was used specifically to populate a Kafka topic, printing different log messages to `stdout` compared to the one seen in a Kafka container when the brokers were getting started. That said, I removed the function that contained these lines of code as populating a topic turned out to be unnecessary for testing.